### DISCUSSION TITLE
How to resize windows in monadtall layout?

### DISCUSSION THREAD

**Discussion by AbdelrhmanUZaki** [qtile/qtile#4749] [Q&A]
Resizing works fine in columns layout but not in monadtall layout

I tried this solution  https://github.com/qtile/qtile/discussions/3736  but didn't work as expected as in columns layout

**Comment by elParaguayo** [comment]
What do you mean "didn't work as expected"?

What command were you running?

**Comment by AbdelrhmanUZaki** [comment]
What I mean that In Column layout, I could resize freely as in video, but in Monaltall it is strange in its resizing, isn't there a way to make resizing in Monadtall as in column layout?

In column I could move in the 4 direction using `hjkl`, but in Monad as a solution it strange

https://github.com/qtile/qtile/assets/99971020/56a65096-95b9-448b-88c8-7f719f6f5750

**Comment by careb0t** [comment]
I would also really like to know the answer to this question if anyone ever figures it out. It seems like the Xmonad layouts are the only ones where I am not able to resize windows. Works fine on all of the other layouts I have tested.
